src/components/AmplitudeAnalytics/constants.ts

```diff
@@ -37,14 +41,21 @@ export enum CUSTOM_USER_PROPERTIES {
   SCREEN_RESOLUTION_HEIGHT = 'screen_resolution_height',
   SCREEN_RESOLUTION_WIDTH = 'screen_resolution_width',
   WALLET_ADDRESS = 'wallet_address',
+  WALLET_NATIVE_CURRENCY_AMOUNT = 'wallet_native_currency_amount',
   WALLET_NATIVE_CURRENCY_BALANCE_USD = 'wallet_native_currency_balance_usd',
-  WALLET_TOKENS_ADDRESSES = 'wallet_tokens_addresses',
-  WALLET_TOKENS_SYMBOLS = 'wallet_tokens_symbols',
   WALLET_TYPE = 'wallet_type',
+  WALLET_USDC_AMOUNT = 'wallet_usdc_amount',
+  WALLET_USDC_BALANCE_USD = 'wallet_usdc_balance_usd',
+  WALLET_WETH_AMOUNT = 'wallet_weth_amount',
+  WALLET_WETH_BALANCE_USD = 'wallet_weth_balance_usd',
 }
 
-export enum CUSTOM_USER_PROPERTY_SUFFIXES {
-  WALLET_TOKEN_AMOUNT_SUFFIX = '_token_amount',
+const ETH = nativeOnChain(1)
```

Comment:
is this and USDC not stored elsewhere as a constant?

Reply:
no they arent right now